charts/victoria-metrics-k8s-stack/README.md.gotmpl

```diff
@@ -14,45 +15,82 @@
 * [Values](#Parameters)
 
 
-# Prerequisites
-
-* Install the follow packages: ``git``, ``kubectl``, ``helm``, ``helm-docs``. See this [tutorial](../../REQUIREMENTS.md).
+# Overview
+This chart is an All-in-one solution to start monitoring kubernetes cluster.
+It installs multiple dependency charts like [grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana), [node-exporter](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-node-exporter), [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics) and [victoria-metrics-operator](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-operator).
+Also it installs Custom Resources like [VMSingle](https://docs.victoriametrics.com/operator/quick-start.html#vmsingle), [VMCluster](https://docs.victoriametrics.com/operator/quick-start.html#vmcluster), [VMAgent](https://docs.victoriametrics.com/operator/quick-start.html#vmagent), [VMAlert](https://docs.victoriametrics.com/operator/quick-start.html#vmalert).
 
-* Add dependency chart repositories
+To enable metrics collection for kubernetes this chart installs multiple scrape configurations for kuberenetes components like kubelet and kube-proxy, etc. Metrics collection is done by [VMAgent](https://docs.victoriametrics.com/operator/quick-start.html#vmagent). So if want to ship metrics to external VictoriaMetrics database you can disable VMSingle installation by setting `vmsingle.enabled` to `false` and setting `vmagent.vmagentSpec.remoteWrite.url` to your external VictoriaMetrics database.
 
-```console
-helm repo add grafana https://grafana.github.io/helm-charts
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts 
-helm repo update
-```
-
-* PV support on underlying infrastructure.
+This chart also installs bunch of dashboards and recording rules from [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) project.
 
+![Overview](img/k8s-stack-overview.png)
 
-# Chart Details
 
-This chart will do the following:
+# Configuration
 
-* Rollout victoria metrics operator, grafana, prometheus-node-exporter and kube-state-metrics helm charts
-* Create VMSingle, VMAgent, VMAlert, VMAlertManager CRDs 
-* Configure Grafana for VictoriaMetrics datasource
-* Configure ServiceScrapes for Node Exporter, kube-state-metrics and various kubernetes metrics
-* Install Grafana dashboards for kubernetes monitoring based on [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus)
+Configuration of this chart is done through helm values. 
```

Comment:
# Upgrade guide

 Update this section with migration guide for node exporter and mention other import changes for values.yaml

Reply:
sure. just wanted to finish with changes before updating 'upgrade' steps